.github/labeler.yml

```diff
@@ -0,0 +1,98 @@
+'pkg:aa':
+  - changed-files:
+      - any-glob-to-any-file:
+          - 'packages/aa/**'
+
+'pkg:allow-scripts':
+  - changed-files:
+      - any-glob-to-any-file:
+          - 'packages/allow-scripts/**'
+
+'pkg:browserify':
```

Comment:
(unsure) how about we use actual package names for these? a bit longer, but maybe mode consistent.

Reply:
fair enough

Reply:
guess it's fine if we leave the prefix in. ok

Reply:
removed `endomoat` entry; we can add it later once merged